res2df/equil.py

```diff
@@ -342,28 +342,28 @@ def equil_main(args) -> None:
 
 
 def equil_reverse_main(args) -> None:
-    """Entry-point for module, for command line utility for CSV to Eclipse"""
-    logger = getLogger_ecl2csv(  # pylint: disable=redefined-outer-name
+    """Entry-point for module, for command line utility for CSV to resdata"""
+    logger = getLogger_res2csv(  # pylint: disable=redefined-outer-name
         __name__, vars(args)
     )
     equil_df = pd.read_csv(args.csvfile)
     logger.info("Parsed %s", args.csvfile)
-    inc_string = df2ecl(equil_df, keywords=args.keywords)
+    inc_string = df2res(equil_df, keywords=args.keywords)
     common.write_inc_stdout_file(inc_string, args.output)
 
 
-def df2ecl(
+def df2res(
     equil_df: pd.DataFrame,
     keywords: Optional[List[str]] = None,
     comments: Optional[Dict[str, str]] = None,
     withphases: bool = False,
     filename: Optional[str] = None,
 ) -> str:
-    """Generate Eclipse include strings from dataframes with
+    """Generate resdata include strings from dataframes with
```

Comment:
`include file format` or `deck format` instead of `resdata include strings` and link to the corresponding glossary term.

Reply:
Formulated slightly differently, as I do think this generates string contents of include files:
```
    """Generate string contents of :term:`include files <include file>`
    from dataframes with solution (EQUIL, RSVD++) data.
"""
```